packages/flutter/lib/src/widgets/image.dart

```diff
@@ -44,6 +45,15 @@ ImageConfiguration createLocalImageConfiguration(BuildContext context, { Size si
   );
 }
 
+/// Prefetches an image into the image cache.
+Future<Null> prefetchImage(ImageProvider provider, BuildContext context, { Size size }) {
```

Comment:
ImageProvider is at the wrong layer; this function has to take a BuildContext to do the right thing.

It's a peer to another global function that does similar things.